### PR TITLE
Telemetry events added for web worker and request sender

### PR DESCRIPTION
**Pull request by haonliu**
#### Description of changes

- Track scan urls added for processing events in web workers.
- ``ScanUrlsAddedForProcessing`` events will be sent per ``OnDemandPageScanBatchRequest`` document processed. 

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts [hunk 1]
```diff
@@ -40,6 +42,7 @@ export class OnDemandDispatcher {
                 itemCount = response.item.length;
                 if (itemCount > 0) {
                     await this.sender.sendRequestToScan(response.item);
+                    queuedItemCount += itemCount;
                     this.logger.logInfo(`[Sender] Queued ${itemCount} scan requests to the queue`);
```
Comment (lamaks) [inline packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts]: can we move event here to have detailed monitoring

Comment (haonliu) [reply packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts]: good point. done.

File: packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { inject, injectable } from 'inversify';
 import { Logger } from 'logger';
 import { PageScanRequestProvider } from 'service-library';
 import { OnDemandPageScanRequest } from 'storage-documents';
+
```
Comment (lamaks) [inline packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts]: why this space matter in import section?

Comment (haonliu) [reply packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts]: removed

File: packages/logger/src/logger-event.ts [hunk 3]
```diff
@@ -6,6 +6,8 @@ export declare type LoggerEvent =
     | 'BatchScanRequestSubmitted'
     | 'ScanRequestSubmitted'
     | 'BatchPoolStats'
+    | 'ScanUrlsAddedForProcessing'
```
[the remark this answers is not in the document]
Comment (haonliu) [reply packages/logger/src/logger-event.ts]: renamed.

**Pull request merged**